lib/properties/bottom.js

```diff
@@ -1,14 +1,20 @@
-'use strict';
+"use strict";
 
-var parseMeasurement = require('../parsers').parseInheritingMeasurement;
+const parseMeasurement = require("../parsers").parseInheritingMeasurement;
 
 module.exports.definition = {
-  set: function (v) {
-    this._setProperty('bottom', parseMeasurement(v));
+  set(v) {
+    if (v === undefined) {
+      return;
+    }
```

Comment:
My reading of the spec is that `undefined` should behave as the string `"undefined"`. In most cases this will fail to parse and thus do nothing, but in some cases it probably has an effect?

Reply:
Done